python/llm/example/simple.py

```diff
@@ -0,0 +1,5 @@
+from bigdl.llm.ggml.model.llama import Llama
+
+model = Llama("model/ggml/gpt4all-model-q4_0.bin")
```

Comment:
shall we use a different filename postfix than llama.cpp, as their formats are not compatible? @TheaperDeng @MeouSker77